tests/main/test_main.py

```
@@ -227,6 +227,20 @@ def test_generate_from_json_input(tmp_path: Path, output_dir: Path) -> None:
+@freeze_time("2020-06-19")
+def test_generate_discriminated_union_with_simple_type(output_dir: Path) -> None:
+    run_cli_and_assert(
+        input_path=DATA_PATH
+        / OPEN_API_DEFAULT_TEMPLATE_DIR_NAME
+        / "discriminated_union_simple_type.yaml",
+        output_path=output_dir,
+        expected_path=EXPECTED_OPENAPI_PATH
+        / "default_template"
+        / "discriminated_union_simple_type",
+    )
+    exec(output_dir.joinpath("models.py").read_text(encoding="utf-8"), {})
```

Ruff (0.15.12)

[error] Use of `exec` detected

(S102)